client/styles/_lists.scss

```diff
@@ -5,7 +5,7 @@ ol {
   list-style: none;
 }
 
-ul.with--bullets {
+.with--bullets {
```

Comment:
Since we're building a new component, we should move these styles in that folder and use them the way we're doing for the rest:

```
import styles from ....;
```